sources/packages/backend/apps/api/src/app.students.module.ts

```diff
@@ -76,7 +75,7 @@ import { ATBCIntegrationModule } from "@sims/integrations/atbc-integration";
     StudentAccountApplicationStudentsController,
     EducationProgramOfferingStudentsController,
     RestrictionStudentsController,
-    ATBCStudentController,
+    //ATBCStudentController, // TODO: This is commented as part of #2539 - Suspend any ATBC integration.
```

Comment:
If the controller is removed from the module, I would recommend not removing the decorator in the controller.

Also, I would say not to keep any commented code. Just having the TODO comment will be mor than sufficient to enable the controller when it is required.